pkg/aggregator/sender_test.go

```diff
@@ -145,25 +145,25 @@ func TestGetAndSetSender(t *testing.T) {
 	eventPlatformChan := make(chan senderEventPlatformEvent, 10)
 	testCheckSender := newCheckSender(checkID1, "", senderMetricSampleChan, serviceCheckChan, eventChan, bucketChan, orchestratorChan, eventPlatformChan)
 
-	err := SetSender(testCheckSender, checkID1)
+	err := demux.SetSender(testCheckSender, checkID1)
 	assert.Nil(t, err)
 
-	sender, err := GetSender(checkID1)
+	sender, err := demux.GetSender(checkID1)
 	assert.Nil(t, err)
 	assert.Equal(t, testCheckSender, sender)
 }
 
 func TestGetSenderDefaultHostname(t *testing.T) {
-	// this test IS USING globals (demultiplexerInstance holding the senders)
+	// this test not using anything global
```

Comment:
What about `aggregatorInstance`?

Reply:
It is read from the local demultiplexer L161:

```
	aggregatorInstance := demux.Aggregator()
```